includes/admin/OrderSettings.php

```diff
@@ -695,4 +734,20 @@ private function setDeliveryOptions($deliveryOptions = null): void
             $this->deliveryOptions = WCMYPA_Admin::getDeliveryOptionsFromOrder($this->order, (array) $deliveryOptions);
         }
     }
+
+    /**
+     * @param string $translatedOptionDesignation human understandable string denoting the concerned option
+     */
+    private function showNotWithPickupWarning(string $translatedOptionDesignation): void
```

Comment:
Deze hele functie is niet echt human, gaat alles goed? 😂 Zou je een wat meer human understandable v2 kunnen indienen van de functienaam en parameter?